### DISCUSSION TITLE
Linux support?

### DISCUSSION THREAD

**Discussion by BananaWorks07** [Inori/GPCS4#116] [Q&A] [answered]
Is it possible to add Linux support since it's just vulkan

**Comment by Inori** [comment]
Do not put feature request here.
Go to Discussions page.

**Comment by Inori** [comment, marked as answer]
Yes it is.
GPCS4 is designed with multi platform support in mind.
It's only the platform layer for linux is not implement yet.
I need to focus on core functionality instead of platform support.

**Comment by Fox2Code** [comment]
Wine exists to run Windows programs on Linux, I haven't tested it yet but the `.exe` should work.

If GPCS4 doesn't work on Wine, opening an issue should make the wine team really happy to help.

**Reply by Inori** [reply]
GPCS4 is not likely to work on wine...
But it is natively supported.
There was a linux branch which can be compiled. But I've deleted since it is only can be compiled.

**Reply by Fox2Code** [reply]
Well if Windows GPCS4 does not work on wine, an issue should be made.

I think when I will have more free time I will try to see if it work, and if it doesn't work I will open some issues on Wine bug tracker.

But actually wine is a very mature project and successfully implemented advanced things such as syscall emulation, memory mapping, Direct(X/Input/Draw) and many other things. 

So there is a high chance the emulator works fine on Wine, especially if the emulator use Vulkan or OpenGL as Linux OpenGL/Vulkan and Windows OpenGL/Vulkan are almost the same.

**Reply by iMonZ** [reply]
Hey have you made any progress?

**Reply by Inori** [reply]
well, nealy comilable again under linux......

**Comment by BananaWorks07** [comment]
I mean libwine can come In clutch if needed
